core-bundle/src/Picker/ArticlePickerProvider.php

```diff
@@ -70,6 +74,27 @@ public function getDcaAttributes(PickerConfig $config): array
     {
         $attributes = ['fieldType' => 'radio'];
 
+        if ('article' === $config->getContext()) {
+            $value = $config->getValue();
+            if ($fieldType = $config->getExtra('fieldType')) {
+                $attributes['fieldType'] = $fieldType;
+            }
+
+            if ($source = $config->getExtra('source')) {
+                $attributes['preserveRecord'] = $source;
+            }
+
+            if ($value) {
+                $intval = static function ($val) {
+                    return (int) $val;
+                };
+
+                $attributes['value'] = array_map($intval, explode(',', $value));
```

Comment:
This should be `array_map('\intval' ….`

Reply:
Changed in b1eeebb30a3fa26f5bc1c042e91ed6889585acb1